### PR TITLE
docs: add blog post for existing s3 with cf

### PR DESCRIPTION
**Pull request by iamhopaul123**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: site/content/blogs/release-v125.en.md [hunk 1]
```diff
@@ -34,11 +34,53 @@ Copilot v1.25 brings several new features and improvements:
 ### Import Values From CloudFormation Stacks In Workload Manifests
 
 ## Static Content Delivery With CloudFront
+You can now bring your own S3 bucket to work with CloudFront for faster static content delivery. More native support for bucket management (for example, bucket creation and assets upload) will be included in future releases.
```
Comment (huanjani) [inline site/content/blogs/release-v125.en.md]:
```suggestion
You can now bring your own S3 bucket to work with CloudFront for faster static content delivery. More native support for bucket management (for example, bucket creation and asset upload) will be included in future releases.
```

File: site/content/blogs/release-v125.en.md [hunk 2]
```diff
@@ -34,11 +34,53 @@ Copilot v1.25 brings several new features and improvements:
...
+
+### (Optional) Create an S3 bucket
+If you don't have an existing S3 bucket, use either S3 console/AWS CLI/SDK to create an S3 bucket. Note that for security concern, we strongly recommend to create a private S3 bucket which blocks public access by default.
```
Comment (huanjani) [inline site/content/blogs/release-v125.en.md]:
```suggestion
If you don't have an existing S3 bucket, use the S3 console/AWS CLI/SDK to create an S3 bucket. Note that for security concerns, we strongly recommend creating a private S3 bucket, which blocks public access by default.
```

File: site/content/blogs/release-v125.en.md [hunk 3]
```diff
@@ -34,11 +34,53 @@ Copilot v1.25 brings several new features and improvements:
...
+
+### Configuring CloudFront in env manifest
```
Comment (huanjani) [inline site/content/blogs/release-v125.en.md]:
```suggestion
### Configuring CloudFront in the env manifest
```

File: site/content/blogs/release-v125.en.md [hunk 4]
```diff
@@ -34,11 +34,53 @@ Copilot v1.25 brings several new features and improvements:
...
+
+```yaml
+cdn:
+  static_assets:
+    location: cf-s3-ecs-demo-bucket.s3.us-west-2.amazonaws.com
+    alias: example.com
+    path: static/*
+```
+
+More specifically `location` is the [DNS domain name of the S3 bucket](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/distribution-web-values-specify.html#DownloadDistValuesDomainName). And the static assets will be accessible at `example.com/static/*`.
```
Comment (huanjani) [inline site/content/blogs/release-v125.en.md]:
```suggestion
More specifically, `location` is the [DNS domain name of the S3 bucket](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/distribution-web-values-specify.html#DownloadDistValuesDomainName), and the static assets will be accessible at `example.com/static/*`.
```

File: site/content/blogs/release-v125.en.md [hunk 5]
```diff
@@ -34,11 +34,53 @@ Copilot v1.25 brings several new features and improvements:
...
+        "Condition": {
+            "StringEquals": {
+                "AWS:SourceArn": "arn:aws:cloudfront::111122223333:distribution/EDFDVBD6EXAMPLE"
+            }
+        }
+    }
+}
+```
+
+Note that you can found the CloudFront distribution ID by running `copilot env show --resources`.
```
Comment (huanjani) [inline site/content/blogs/release-v125.en.md]:
```suggestion
You can find the CloudFront distribution ID by running `copilot env show --resources`.
```

File: site/content/blogs/release-v125.en.md [hunk 6]
```diff
@@ -34,11 +34,53 @@ Copilot v1.25 brings several new features and improvements:
...
+
+```yaml
+cdn:
+  static_assets:
+    location: cf-s3-ecs-demo-bucket.s3.us-west-2.amazonaws.com
+    alias: example.com
+    path: static/*
+```
+
+More specifically `location` is the [DNS domain name of the S3 bucket](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/distribution-web-values-specify.html#DownloadDistValuesDomainName), and the static assets will be accessible at `example.com/static/*`.
```
Comment (huanjani) [inline site/content/blogs/release-v125.en.md]:
```suggestion
More specifically, `location` is the [DNS domain name of the S3 bucket](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/distribution-web-values-specify.html#DownloadDistValuesDomainName), and the static assets will be accessible at `example.com/static/*`.
```

**Review by huanjani — APPROVED** [review]
Looks good! Just one tiny nit, feel free to remove the DNM label after you address it.

**Review by Lou1415926 — APPROVED** [review]

**Pull request merged**